ocaml/networkd/lib/network_utils.ml

```diff
@@ -112,7 +112,7 @@ let check_n_run ?(on_error = default_error_handler) ?(log = true) run_func
   try
     Unix.access script [Unix.X_OK] ;
     (* Use the same $PATH as xapi *)
-    let env = [|"PATH=" ^ Sys.getenv "PATH"|] in
+    let env = [|"PATH=" ^ Option.value (Sys.getenv_opt "PATH") ~default:""|] in
```

Comment:
While PATH not existing shouldn't happen, having an empty array would match would better match the xapi environment:
```suggestion
    let env = Option.fold ~none:[||] ~some:(fun p -> [|"PATH=" ^ p|] (Sys.getenv_opt "PATH") in
```